boards/arm/mps2_an521/doc/index.rst

```diff
@@ -380,20 +380,13 @@ The process requires five steps:
 4. Merge the two binaries together and sign them.
 5. Concatenate the bootloader with the signed image blob.
 
-In order to build tfm please refer to `Trusted Firmware M Guide`_.
+To build tfm as secure image, refer to `Trusted Firmware M Guide`_.
```

Comment:
Isn't this redundant @microbuilder ? I mean TF-M is by definition a secure image

Reply:
I guess I was just trying to indicate the distinction between TF-M as the source of the S binary, running in the secure processing environment, contrasted with Zephyr as the non-secure image, and the two images get merged as part of the build process. I can reword this if it doesn't seem intuitive as is, though.

Reply:
Updated in https://github.com/zephyrproject-rtos/zephyr/pull/19985/commits/9f39aeecfed25dffe1f225627f11cacb0424e362